src/data/db_schema.py

```diff
@@ -22,111 +21,7 @@
 class Base(MappedAsDataclass, DeclarativeBase):
     """Base class to subclass from to define the schema"""
 
-
-# TODO: Rename this to something more accurate to what is happening, e.g. HyperdriveTransactions
-class WalletInfo(Base):
-    """Table/dataclass schema for wallet information."""
-
-    __tablename__ = "walletinfo"
-
-    # Default table primary key
-    # Note that we use postgres in production and sqlite in testing, but sqlite has issues with
-    # autoincrement with BigIntegers. Hence, we use the Integer variant when using sqlite in tests
-    id: Mapped[int] = mapped_column(
-        BigInteger().with_variant(Integer, "sqlite"), primary_key=True, init=False, autoincrement=True
-    )
-
-    blockNumber: Mapped[int] = mapped_column(BigInteger, ForeignKey("poolinfo.blockNumber"), index=True)
-    walletAddress: Mapped[Union[str, None]] = mapped_column(String, index=True, default=None)
-    # baseTokenType can be BASE, LONG, SHORT, LP, or WITHDRAWAL_SHARE
-    baseTokenType: Mapped[Union[str, None]] = mapped_column(String, index=True, default=None)
-    # tokenType is the baseTokenType appended with "-<maturity_time>" for LONG and SHORT
-    tokenType: Mapped[Union[str, None]] = mapped_column(String, default=None)
-    tokenValue: Mapped[Union[Decimal, None]] = mapped_column(Numeric, default=None)
-    maturityTime: Mapped[Union[int, None]] = mapped_column(BigInteger().with_variant(Integer, "sqlite"), default=None)
-    sharePrice: Mapped[Union[Decimal, None]] = mapped_column(Numeric, default=None)
-
-
-class WalletDelta(Base):
-    """Table/dataclass schema for wallet information."""
-
-    __tablename__ = "walletdelta"
-
-    # Default table primary key
-    # Note that we use postgres in production and sqlite in testing, but sqlite has issues with
-    # autoincrement with BigIntegers. Hence, we use the Integer variant when using sqlite in tests
-    id: Mapped[int] = mapped_column(
-        BigInteger().with_variant(Integer, "sqlite"), primary_key=True, init=False, autoincrement=True
-    )
-    transactionHash: Mapped[str] = mapped_column(String, ForeignKey("transactions.transactionHash"), index=True)
     blockNumber: Mapped[int] = mapped_column(BigInteger, ForeignKey("poolinfo.blockNumber"), index=True)
```

Comment:
I think this is now saying every db schema table has a field called `blockNumber` that's a foreignKey to poolinfo.blockNumber, which doesn't seem correct.

Reply:
For example, there are tables that don't have a blockNumber at all, e.g., `usermap`, `poolconfig`, etc

Reply:
ya its not right.  i was going for an abstract table here, fixed it.  good catch.